### ISSUE TITLE
Different chords for different languages.

### ISSUE THREAD

**Issue by alex-belost** [CharaChorder/DeviceManager#146]
It would be great to add the ability to have different sets of chords for different languages.

**Comment by pepesanchezb** [comment]
Yes, for sure someone has a file with chords in other languages, please share it
